From bf6cab60e8f09a27c52c020ef8684fdd18efb625 Mon Sep 17 00:00:00 2001
From: Daniel Kahn Gillmor <dkg@fifthhorseman.net>
Date: Fri, 30 Aug 2024 00:18:08 -0400
Subject: [PATCH 2/5] Drop wasm -- not for Debian right now

This would create a dependency on the js feature of getrandom, which
is not in debian at the moment.
---
 Cargo.toml | 5 -----
 1 file changed, 5 deletions(-)

diff --git a/Cargo.toml b/Cargo.toml
index 426337f..7b10977 100644
--- a/Cargo.toml
+++ b/Cargo.toml
@@ -318,8 +318,3 @@ nightly = [
     "num-bigint/nightly",
 ]
 profile = ["gperftools"]
-wasm = [
-    "chrono/wasmbind",
-    "getrandom",
-    "getrandom/js",
-]
-- 
2.45.2

